transformation/plugins/org.jboss.tools.fuse.transformation.editor/src/main/java/org/jboss/tools/fuse/transformation/editor/internal/wizards/TransformTestWizardPage.java

```diff
@@ -484,12 +474,13 @@ private void doStatusUpdate() {
             }
         }
 
-        if (fPackageStatus.getCode() == StatusInfo.ERROR && (fPackageStatus.getMessage() == null
-                || fPackageStatus.getMessage().trim().isEmpty())) {
-            // override this particular case, since the default package is ok, though not great
-            fPackageStatus = new StatusInfo(NONE, null);
-        } else if (fPackageStatus.getCode() == StatusInfo.WARNING && fPackageStatus.getMessage() != null
-                && fPackageStatus.getMessage().contains("default package is discouraged")) { //$NON-NLS-1$
+        boolean defaultPkg = fPackageStatus.getCode() == StatusInfo.ERROR && (fPackageStatus.getMessage() == null
+            				 || fPackageStatus.getMessage().trim().isEmpty());
+        if (!defaultPkg) {
+        	defaultPkg = fPackageStatus.getCode() == StatusInfo.WARNING && fPackageStatus.getMessage() != null
+                		 && fPackageStatus.getMessage().contains("default package is discouraged"); //$NON-NLS-1$
```

Comment:
I think it is best to use the internal JDT class, otherwise it is not working when a user is using the Fuse Tooling with a Locale translated different than English.